packages/flutter/lib/src/material/dialog.dart

```diff
@@ -469,12 +473,23 @@ class AlertDialog extends StatelessWidget {
       }
     }
 
+    // The paddingScaleFactor is used to adjust the edge padding of Dialog
```

Comment:
Not clear what "edge padding" means here.

Reply:
replaced with just padding